app/models/concerns/clone/session.rb

```diff
@@ -38,7 +38,7 @@ def outcome_questions
     @outcome_questions ||= Question.unscoped
             .includes(:question_group)
             .where(question_groups: { session_id: outcome.id })
-            .order('question_groups.position ASC', 'questions.position ASC')
+            .order('question_groups.position ASC', 'question.position ASC')
```

Comment:
Why isn't it left as `questions`? Is it referring to the table name or am I missing something?

Reply:
TGH, I don't remember exactly why I did it. I double-checked this and with `questions` works so I undid that.